etc/answers-react-components.api.md

```diff
@@ -357,8 +379,8 @@ declare namespace Filters {
         CollapsibleLabelProps,
         CollapsibleSection,
         CollapsibleSectionProps,
-        Facets,
-        FacetsProps,
+        Facets_2 as Facets,
+        FacetsProps_2 as FacetsProps,

```

Comment:
can we remove the Facets exports from the Filters namespace so we're not exposing both to the user?

Reply:
HierarchicalFacet, at least in our tests, still depends on it

Reply:
even in that case I think we can remove the export in Filters/index.ts? because in the tests we can just change to import it directly from src/components/Filters/Facets.tsx instead of src/components/Filters

Reply:
sure

Reply:
actually, thinking about it more, if we use the Facets compound component in HierarchicalFacets, then other people would need it too. so until we make a baked-in component for that we should probably export it to users even if it means our docs are a little messed up in the meantime

Reply:
I believe we will have changes for a higher level HierarchicalFacets but I can add this back for now. Our docs wouldn't be messed up if you look at the documentation pages on repo since this is exported under a namespace 'Filters' - it just looks like it from the files here

Reply:
ah, got it!